samples/drivers/led_lp3943/prj.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_SYS_LOG=y
+CONFIG_BOOT_BANNER=y
+
+CONFIG_I2C=y
+
+CONFIG_LED=y
+CONFIG_LP3943=y
+CONFIG_LP3943_I2C_MASTER_DEV_NAME="I2C_3"
```

Comment:
This line is board-specific -- how about moving it to boards/96b_neonkey.conf so others can customize it to their needs in their own board files?

Reply:
Sure, will move it to board specific config file.